airgun/views/contentviewfilter.py

```diff
@@ -0,0 +1,343 @@
+    )
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+                breadcrumb_loaded
+                and self.breadcrumb.locations[0] == 'Content Views'
+                and self.breadcrumb.read() == 'Yum Filters'
+        )
+
+
+class CreateYumFilterView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+
+    name = TextInput(id='name')
+    content_type = Select(id='type')
+    inclusion_type = Select(id='inclusion')
+    description = TextInput(id='description')
+
+    save = Text('.//button[contains(@ng-click, "handleSave()")]')
+    cancel = Text('.//button[@ng-click="handleCancel()"]')
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+            breadcrumb_loaded
+            and self.breadcrumb.locations[0] == 'Content Views'
+            and self.breadcrumb.read() == 'Create Yum Filter'
+        )
+
+
+class EditYumFilterView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+    filter_type = Text('//header/small')
+
+    content_tabs = ConditionalSwitchableView(reference='filter_type')
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+            breadcrumb_loaded
+            and len(self.breadcrumb.locations) > 3
+            and self.breadcrumb.locations[2] == 'Yum Filters'
+            and self.breadcrumb.read() != 'Create Yum Filter'
+        )
+
+    @View.nested
+    class details(SatSecondaryTab):
+        name = CVFEditableEntry(name='Name')
+        description = CVFEditableEntry(name='Description')
+
+    @content_tabs.register(lambda filter_type: filter_type.endswith('RPM'))
+    class rpm_filter(View):
+
+        @View.nested
+        class rpms(SatSecondaryTab, SearchableViewMixin):
+            TAB_NAME = 'RPMs'
+
+            exclude_no_errata = Checkbox(
+                locator=".//input[@type='checkbox']"
+                        "[@ng-model='filter.original_packages']"
+            )
+            add_rule = Text(".//button[@ng-click='addRule()']")
+            remove_rule = Text(".//button[@ng-click='removeRules(filter)']")
+            table = SatTable(
+                locator='//table',
+                column_widgets={
+                    0: Checkbox(locator=".//input[@type='checkbox']"),
+                    'RPM Name': TextInput(locator='.//input'),
+                    'Architecture': TextInput(locator='.//input'),
+                    'Version': CVFRuleVersion(),
+                    4: CVFRuleActions(),
+                },
+            )
+
+    @content_tabs.register(lambda filter_type: filter_type.endswith('Errata'))
+    class errata_filter(AddRemoveResourcesView):
+
+        @View.nested
+        class AddTab(SatSecondaryTab):
```

Comment:
we need a little bit more descriptive name here to make strict difference between errata types

Reply:
Firstly, that's the same name as in `AddRemoveResourcesView` as i'm overriding that tab so i can't use different name (unless we decide to completely copy&paste all the code, including List/Remove tab which is not different from `AddRemoveResourcesView` implementation). Secondly, when tab name is just 'Add', how can i name it more descriptively? :)